pkg/api/annotations.go

```diff
@@ -48,12 +49,13 @@ func (e *CreateAnnotationError) Error() string {
 	return e.message
 }
 
-func PostAnnotation(c *middleware.Context, cmd dtos.PostAnnotationsCmd) Response {
+func PostAnnotation(c *middleware.Context, cmd dtos.PostAnnotationsCmd) {
```

Comment:
Its currently kinda mixed between using `Response` or not. 

Last time I and @torkelo talked about this we agreed on using Response since its easy to forget to return after using `c.JsonApiErr`. So using Response is preferred. 

This is the way to do custom return types using `Response` https://github.com/grafana/grafana/blob/master/pkg/api/datasources.go#L149

Reply:
Fixed